packages/solid/src/index.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Copyright 2024 Design Barn Inc.
```

Comment:
> Copyright 2024 Design Barn Inc.


This notice can be removed

Reply:
Seems like dotlottie is using some kind of eslint plugin
https://www.npmjs.com/package/eslint-plugin-header

Reply:
I think we should disable this rule from the eslint config file, since it's an open-source project under the MIT license

Reply:
Resolved on https://github.com/LottieFiles/dotlottie-web/pull/234/commits/3b153a4f532a77713bcf3676eeb510f4b6705501